Sub/Src/ComputerVision/libs/pose_calculation.py

```diff
@@ -0,0 +1,126 @@
+        self.distort_var_5 = float(self.param_serv.get_param("Vision/solvePnP/distortion_matrix/k3"))
+
+        self.distortion_matrix = np.array([[self.distort_var_1, self.distort_var_2, self.distort_var_3, self.distort_var_4, self.distort_var_5]])
+        #print(self.distortion_matrix)
+
+        label = self.detection[0]
+
+        if(label == b'Dice 1' or label == b'Dice 2' or label == b'Dice 3' or label == b'Dice 4' or label == b'Dice 5' or label == b'Dice 6'):
+            self.min_coordinate = float(self.param_serv.get_param("Vision/Coordinates/dice/topleft"))
+            self.center_coordinate = float(self.param_serv.get_param("Vision/Coordinates/dice/middle"))
+            self.max_coordinate = float(self.param_serv.get_param("Vision/Coordinates/dice/bottomright"))
+            self.three_dim_points = np.array([[self.min_coordinate, self.min_coordinate, self.min_coordinate],
+                                              [self.center_coordinate, self.min_coordinate, self.min_coordinate],
+                                              [self.max_coordinate, self.min_coordinate, self.min_coordinate],
+                                              [self.min_coordinate, self.center_coordinate, self.min_coordinate],
+                                              [self.center_coordinate, self.center_coordinate, self.min_coordinate],
+                                              [self.max_coordinate, self.center_coordinate, self.min_coordinate],
+                                              [self.min_coordinate, self.max_coordinate, self.min_coordinate],
+                                              [self.center_coordinate, self.max_coordinate, self.min_coordinate],
+                                              [self.max_coordinate, self.max_coordinate, self.min_coordinate]])
+
+            self.two_dim_points = np.array([[(self.x_coordinate - (0.5 * self.width)), (self.y_coordinate - (0.5 * self.height))],
+                                            [(self.x_coordinate), (self.y_coordinate - (0.5 * self.height))],
+                                            [(self.x_coordinate + (0.5 * self.width)), (self.y_coordinate - (0.5 * self.height))],
+                                            [(self.x_coordinate - (0.5 * self.width)), (self.y_coordinate)],
+                                            [(self.x_coordinate), (self.y_coordinate)],
+                                            [(self.x_coordinate + (0.5 * self.width)), (self.y_coordinate)],
+                                            [(self.x_coordinate - (0.5 * self.width)), (self.y_coordinate + (0.5 * self.height))],
+                                            [(self.x_coordinate), (self.y_coordinate + (0.5 * self.height))],
+                                            [(self.x_coordinate + (0.5 * self.width)), (self.y_coordinate + (0.5 * self.height))]])
+
+        else if(label == b'Gate Arm'):
```

Comment:
You need to add `pass` here if it doesn't do anything, or else there will be a syntax error